### PR TITLE
Documentation for plausibleBeforeDeath and plausibleStartBeforeEnd

### LINKED ISSUE #541: Add check documentation page for 2 new temporal plausibility checks
plausibleBeforeDeath and plausibleStartBeforeEnd need check-level docs pages.  Templates are already available [here](https://github.com/OHDSI/DataQualityDashboard/blob/main/vignettes/checks/plausibleBeforeDeath.Rmd) and [here](https://github.com/OHDSI/DataQualityDashboard/blob/main/vignettes/checks/plausibleStartBeforeEnd.Rmd).

### PR DESCRIPTION
**Pull request by MaximMoinat**
Fixes #541. Needs to be rendered with pkgdown after review.

### CODE REVIEW

**Comment by MaximMoinat** [comment]
@katy-sadowski, some thoughts while writing the docs:
For `plausibleStartBeforeEnd`, I am doubting whether it is CDM Convention or Critical. I argue the latter, as end dates before start dates creates negative durations which might break analyses. This would also mean we should lower the threshold to 0%, always failing if any records affected.

For `plausibleBeforeDeath`, we have a 60 days grace period. I've explained it as that administrative events are expected until the next month. Or was this an arbitrary choice.

**Review by katy-sadowski — COMMENTED** [review]
Woohooo thanks so much Maxim!

Re: the 60 day buffer - I found the explanation on the Themis docs: https://ohdsi.github.io/Themis/obs_periods_for_ehr.html#observation-period-end-date.  I do wonder if this warrants its own general convention for "How to handle events dating after death"?  @clairblacketer  what do you think?  [PS - super exciting to be able to search for and find that on the website 😄 ]

Re: start before end - interesting question.  I am torn, because this could cause really serious problems in some cases, maybe even errors in code, so that would be an argument for Fatal.  On the other hand I think the intention of Fatal was really database conformance level stuff.  @clairblacketer what do you think?

File: vignettes/checks/plausibleStartBeforeEnd.Rmd [hunk 1]
```diff
@@ -14,33 +14,50 @@ output:
...
-- *Numerator*: 
-- *Denominator*:
-- *Related CDM Convention(s)*:
-- *CDM Fields/Tables*:
-- *Default Threshold Value*:
+- *Numerator*: The number of records where date in **cdmFieldName** is after the date in **plausibleStartBeforeEndFieldName**.
+- *Denominator*: The total number of records with a non-null start and non-null end date value
+- *Related CDM Convention(s)*: -Not linked to a convention-
+- *CDM Fields/Tables*: This check runs on all date and datetime fields that have a start and end date in the same table. It also runs on the `cdm_source` table, comparing `source_release_date` is before `cdm_release_date`.
+- *Default Threshold Value*: 1% (except for vocabulary and cdm_source tables, where it is 0%)
```
[the remark this answers is not in the document]
Comment (katy-sadowski) [reply vignettes/checks/plausibleStartBeforeEnd.Rmd:33]: [regardless of fatal vs. convention, i think we should 👍 ]

[the remark this answers is not in the document]
Comment (MaximMoinat) [reply vignettes/checks/plausibleStartBeforeEnd.Rmd:33]: It is already actually, so will add that to the description here

**Comment by MelaniePhilofsky** [comment]
We do have a ratified convention for events after death. It needs to be documented in the Themis convention library. Right now it is just a ratified and closed in the Themis issues https://github.com/OHDSI/Themis/issues/42
I will reopen it as a "needs documentation"

**Review by katy-sadowski — APPROVED** [review]
Thank you @MaximMoinat !

Maybe we can discuss Fatal vs Convention for StartBeforeEnd at the next DQ call.  Can merge in for now and update if we decide to change it later :)

**Pull request merged**